api/handler/multipart_upload.go

```diff
@@ -443,6 +443,12 @@ func (h *handler) CompleteMultipartUploadHandler(w http.ResponseWriter, r *http.
 			h.logAndSendError(w, "could not translate acl of completed multipart upload to ast", reqInfo, err, additional...)
 			return
 		}
+
+		if sessionTokenSetEACL, err = getSessionTokenSetEACL(r.Context()); err != nil {
```

Comment:
Is there any corresponding bug?

Reply:
Yes, we have session token, and pass session token, but this token is always empty. 
It leads to an infinity wait for the `setEacl` result. But it fails, because the gate isn't a container owner, thus we have a session, but it's empty